0016-pre-shared-key/0016-pre-shared-key.md

```diff
@@ -0,0 +1,57 @@
+# Pre Shared Key
+
+## Data Format
+
+The PSK details are divided into two HTTP-style request objects. Each of the
+requests starts with `PSK/1.0` and then `INSECURE` or `SECURE`.  The requests
+are made of a list of headers, followed by data. For simplicity, duplicate
+headers are not allowed. 
+
+### Outer Request
+
+The outer request is formatted as below:
+
+```http
+PSK/1.0 INSECURE
```

Comment:
> If the intention is that "SECURE" or "INSECURE" are only meant to indicate whether the request is an inner or outer object, and we expect there to always be exactly one of each

That's correct. I think @sharafian's intention was to discourage people from putting sensitive data in the outer (unencrypted) section